_misc/sanity_utils.py

```diff
@@ -0,0 +1,159 @@
+import sys
+import fnmatch
+import os
+import posixpath
+import re
+from ast import iter_fields, AST, Attribute, Name
+
+if sys.version_info[0] == 3:
+    string_types = (str,)
```

Comment:
Linear curve fitting over Python's major versions and their release dates seems to say we're hitting Python 4 around 1st June 2020, so I don't think we need to worry just yet.